tests/acceptance/guid-node/addons-test.ts

```diff
@@ -0,0 +1,342 @@
+        const s3AuthAccount = server.create('authorized-storage-account', {
+            displayName: 'An Amazon S3 account',
+            authorizedCapabilities: ['write'],
+            externalStorageService: s3,
+            accountOwner: userRef,
+            credentialsAvailable: false,
+        });
+        const otherS3AuthAccount = server.create('authorized-storage-account', {
+            displayName: 'Another Amazon S3 account',
+            authorizedCapabilities: ['write'],
+            externalStorageService: s3,
+            accountOwner: userRef,
+            credentialsAvailable: false,
+        });
+
+        // create configured accounts
+        server.create('configured-storage-addon', {
+            displayName: 'My Box Account',
+            rootFolder: 'boxroot',
+            baseAccount: boxAuthAccount,
+            externalStorageService: box,
+            accountOwner: userRef,
+            authorizedResource: nodeRef,
+        });
+        const s3AccountsDisplayNamesAndRootFolders = [{
+            displayName: 'My Box Account',
+            rootFolder: 's3root',
+        }, {
+            displayName: 'My AmazonS3 Account',
+            rootFolder: 'others3root',
+        }];
+        server.create('configured-storage-addon', {
+            displayName: s3AccountsDisplayNamesAndRootFolders[0].displayName,
+            rootFolder: s3AccountsDisplayNamesAndRootFolders[0].rootFolder,
+            baseAccount: s3AuthAccount,
+            externalStorageService: s3,
+            accountOwner: userRef,
+            authorizedResource: nodeRef,
+        });
+        server.create('configured-storage-addon', {
+            displayName: s3AccountsDisplayNamesAndRootFolders[1].displayName,
+            rootFolder: s3AccountsDisplayNamesAndRootFolders[1].rootFolder,
+            baseAccount: otherS3AuthAccount,
+            externalStorageService: s3,
+            accountOwner: userRef,
+            authorizedResource: nodeRef,
+        });
+
+        const url = `/${node.id}/addons`;
+        await visit(url);
+        assert.equal(currentURL(), url, `We are on ${url}`);
+        await percySnapshot('Acceptance | guid-node/addons | Editing configured addons | all addons tab');
+
+        // check additonal storage providers
+        assert.dom('[data-test-addon-card-configure]')
+            .exists({ count: 2 }, '2 providers with configured addons are present');
+        assert.dom('[data-test-addon-card="Amazon S3"]').containsText('Configure', 'Amazon S3 addon card is present');
+        assert.dom('[data-test-addon-card="Box"]').containsText('Configure', 'Box addon card is present');
+
+        // check connected accounts tab
+        await click('[data-test-addons-tab-connected-accounts]');
+        await percySnapshot('Acceptance | guid-node/addons | Editing configured addons | connected accounts tab');
+        assert.dom('[data-test-addon-card]').exists({ count: 2 }, '2 providers with accounts are present');
```

Comment:
Just an aside, I vaguely recall that someone (either Mark, Abram, or both) suggested that we automatically switch to this tab on pageload if there are connected accounts. No need to change anything right now, but that may be coming in UAT.